### DISCUSSION TITLE
date range picker doesn't update on the first click.

### DISCUSSION THREAD

**Discussion by sinafath** [adobe/react-spectrum#4859] [Q&A]
I like my range picker calendar to be updated when I click on the start point. I think It should  be better that date field to be updated even though I didn't select the end point
![Screenshot 2023-08-03 043621](https://github.com/adobe/react-spectrum/assets/83284138/2a9f4cf4-70fb-462c-9211-c7b6f8d06eba)

an example from [mui](https://codesandbox.io/s/pqgggp?file=/Demo.tsx) if you select start date  the upper field will be updated

**Comment by carlesba** [comment]
Any update on this? Is it possible at all?
